src/region/RegionAnnotations.tsx

```diff
@@ -60,10 +60,15 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
         setStatus(CreatorStatus.started);
     };
 
-    handleStop = (shape: Rect): void => {
+    handleStop = (shape: Rect | null): void => {
         const { location, setStaged, setStatus } = this.props;
-        setStaged({ location, shape });
-        setStatus(CreatorStatus.staged);
+        if (shape) {
+            setStaged({ location, shape });
+            setStatus(CreatorStatus.staged);
+        } else {
+            // Do we need to setStaged back to CreatorStatus.init?
+            setStatus(CreatorStatus.reset);
```

Comment:
I think introducing a new status is risky. We've already used `CreatorStatus.init` in a lot of places, and you have to make sure in those places, the new `reset` also works well.
How about we still use `init` and add `'init'` input in Preview FSM?

Reply:
Yes that makes sense, will try that approach instead.

Reply:
Any thoughts on if we need to call `setStaged` method in the else case?

Reply:
Yeah, maybe something like

Reply:
Yeah maybe something like:
```
setStaged(shape? { location, shape } : null);
setStatus(shape? CreatorStatus.staged : CreatorStatus.init);
```